src/game.rs

```diff
@@ -17,34 +17,49 @@ pub mod map;
 pub mod market;
 pub mod shards;
 
-game_map_access! {
-    /// See [http://docs.screeps.com/api/#Game.constructionSites]
-    ///
-    /// [http://docs.screeps.com/api/#Game.constructionSites]: http://docs.screeps.com/api/#Game.constructionSites
-    (construction_sites, objects::ConstructionSite, Game.constructionSites),
-    /// See [http://docs.screeps.com/api/#Game.creeps]
-    ///
-    /// [http://docs.screeps.com/api/#Game.creeps]: http://docs.screeps.com/api/#Game.creeps
-    (creeps, objects::Creep, Game.creeps),
-    /// See [http://docs.screeps.com/api/#Game.flags]
-    ///
-    /// [http://docs.screeps.com/api/#Game.flags]: http://docs.screeps.com/api/#Game.flags
-    (flags, objects::Flag, Game.flags),
-    // TODO: See [http://docs.screeps.com/api/#Game.resources]
-    ///
-    /// [http://docs.screeps.com/api/#Game.resources]: http://docs.screeps.com/api/#Game.resources
-    /// See [http://docs.screeps.com/api/#Game.rooms]
-    ///
-    /// [http://docs.screeps.com/api/#Game.rooms]: http://docs.screeps.com/api/#Game.rooms
-    (rooms, objects::Room, Game.rooms),
-    /// See [http://docs.screeps.com/api/#Game.spawns]
-    ///
-    /// [http://docs.screeps.com/api/#Game.spawns]: http://docs.screeps.com/api/#Game.spawns
-    (spawns, objects::StructureSpawn, Game.spawns),
-    /// See [http://docs.screeps.com/api/#Game.structures]
-    ///
-    /// [http://docs.screeps.com/api/#Game.structures]: http://docs.screeps.com/api/#Game.structures
-    (structures, objects::Structure, Game.structures)
+/// See [http://docs.screeps.com/api/#Game.constructionSites]
+///
+/// [http://docs.screeps.com/api/#Game.constructionSites]: http://docs.screeps.com/api/#Game.constructionSites
+pub mod construction_sites {
+    game_map_access!(objects::ConstructionSite, Game.constructionSites);
+}
+
+/// See [http://docs.screeps.com/api/#Game.creeps]
+///
+/// [http://docs.screeps.com/api/#Game.creeps]: http://docs.screeps.com/api/#Game.creeps
+pub mod creeps {
+    game_map_access!(objects::Creep, Game.creeps);
+}
+
+/// See [http://docs.screeps.com/api/#Game.flags]
+///
+/// [http://docs.screeps.com/api/#Game.flags]: http://docs.screeps.com/api/#Game.flags
+pub mod flags {
+    game_map_access!(objects::Flag, Game.flags);
+}
+
+// TODO: See [http://docs.screeps.com/api/#Game.resources]
+///
```

Comment:
Not needed

Reply:
I didn't change it in this PR, but it should be a double-`//` comment. The line below this one mentions `Game.resources`, and I assume it was intended as part of the TODO.